lib/presentation/community/chat_screen/pages/community_chat_screen_page.dart

```diff
@@ -85,49 +90,51 @@ class CommunityChatScreenPage extends StatelessWidget {
       appBar: CustomAppBar(
-        title: communityChatData.title,
-        trailingWidget: IconButton(
-          key: moreAppBarKey,
-          icon: const Icon(Icons.more_vert),
-          onPressed: () {
-            ScaffoldMessenger.of(context)
-                .showSnackBar(const SnackBar(content: Text(comingSoonText)));
-          },
-        ),
+        title: widget.communityChatData.title,
+        trailingWidget: const MoreMenuDropDown(),
       ),
       body: Padding(
         padding: const EdgeInsets.all(20),
         child: Stack(
           children: <Widget>[
-            ListView.builder(
-              itemCount: messages.length,
-              shrinkWrap: true,
-              padding: const EdgeInsets.only(top: 10, bottom: 10),
-              itemBuilder: (BuildContext context, int index) {
-                return Container(
-                  padding: const EdgeInsets.only(
-                    left: 14,
-                    right: 14,
-                    top: 10,
-                    bottom: 10,
-                  ),
-                  child: Align(
-                    alignment: messages[index] is TimeClassificationWidget
-                        ? Alignment.topCenter
-                        : messages[index] is ReceivedMessageItem
-                            ? Alignment.topLeft
-                            : Alignment.topRight,
-                    child: messages[index],
-                  ),
-                );
-              },
+            Padding(
+              padding: const EdgeInsets.only(bottom: 50),
+              child: ListView.builder(
+                itemCount: messages.length,
+                shrinkWrap: true,
+                padding: const EdgeInsets.only(top: 10, bottom: 10),
+                itemBuilder: (BuildContext context, int index) {
+                  return Container(
+                    padding: const EdgeInsets.only(
+                      left: 14,
+                      right: 14,
+                      top: 10,
+                      bottom: 10,
+                    ),
+                    child: Align(
+                      alignment: messages[index] is TimeClassificationWidget
+                          ? Alignment.topCenter
+                          : messages[index] is ReceivedMessageItem
+                              ? Alignment.topLeft
+                              : Alignment.topRight,
+                      child: messages[index],
+                    ),
+                  );
+                },
+              ),
             ),
             Align(
               alignment: Alignment.bottomCenter,
               child: MessageInput(
                 onChanged: (String value) {
-                  ScaffoldMessenger.of(context)
-                      .showSnackBar(snackbar(content: value));
+                  newMessage = value;
+                },
+                onTap: () {
+                  ScaffoldMessenger.of(context).showSnackBar(
```

Comment:
```suggestion
                ScaffoldMessenger.of(context)
                        ..hideCurrentSnackBar()
                        ..showSnackBar(
```

To ensure that any existing snackbar is closed first